packages/flutter/test/widgets/texture_test.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' as ui;
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+void main() {
+  testWidgets('Texture with default FilterQuality', (WidgetTester tester) async {
```

Comment:
Maybe also add tests to ensure that the filter quality is sent to the RenderObject/Layer from the Widget.

Reply:
OK, i will add tests after https://github.com/flutter/engine/pull/18814 is merged

Reply:
Add TextureBox and TextureLayer related tests.